app/src/test/java/com/lastaoutdoor/lasta/data/auth/AuthRepositoryImplTest.kt

```diff
@@ -251,4 +251,14 @@ class AuthRepositoryImplTest {
       }
     }
   }
+
+  @Test
+  fun `deleteAccount works emits loading then success`() = runTest {
```

Comment:
Here you are not testing that it emits loading at all

Reply:
+1